README.md

```diff
@@ -296,9 +296,26 @@ You can use `config.getRemote()` and `config.setRemote()` for
 
 `npm test`
 
+## Best Practices
+
+Zero-config is designed to help you structure your config 
+files to support a number of production concerns. These best
+ practices reflect our approach and some of the reasons we 
+ designed Zero-config as we did.
+
+ - Most configuration should live in `common.json`
```

Comment:
A baseline file should have the main config I agree. We can either make common.js our primary production file or do some other change. I guess that would make it a breaking change though.

Suggestions on rephrasing this?